### PR TITLE
Add order first to grid

### PR DESCRIPTION
**Pull request by lukefrake**
Fix : #23400

### CODE REVIEW

File: docs/4.0/layout/grid.md [hunk 1]
```diff
@@ -515,7 +515,7 @@ In addition to column clearing at responsive breakpoints, you may need to reset
 
 ### Order classes
 
-Use `.order-` classes for controlling the **visual order** of your content. These classes are responsive, so you can set the `order` by breakpoint (e.g., `.order-1.order-md-2`). Includes support for `1` through `12` across all five grid tiers.
+Use `.order-` classes for controlling the **visual order** of your content. These classes are responsive, so you can set the `order` by breakpoint (e.g., `.order-1.order-md-2`). Includes support for `1` through `12` across all five grid tiers. You also set the class of an element to be `first` which will give it negative order and so put it before unordered elements.
```
Comment (XhmikosR) [inline docs/4.0/layout/grid.md]: You **can** also set the class of an element to be first with `.order-first`, which will give it a negative order and so place it before the unordered elements.

**Comment by mdo** [comment]
Updated the docs here to fix @XhmikosR's feedback by revamping them. We're introducing a new class name, which is best served by a new code example rather than mixing the two.

**Review by mdo — APPROVED** [review]

**Comment by lukefrake** [comment]
@mdo that is much clearer, I considered doing that but didn't want to bloat the docs. LGTM though

Do you want me to revert the order-first in the code element above or are you happy to keep that?

**Review by XhmikosR — APPROVED** [review]

**Comment by mdo** [comment]
@lukefrake Just took care of that, good catch.

**Comment by lukefrake** [comment]
@mdo no problem, happy to help!

**Pull request merged**